pkg/api/api.go

```diff
@@ -149,6 +149,16 @@ func (p Padding) MarshalJSON() ([]byte, error) {
 	return []byte(s), nil
 }
 
+// UnmarshalJSON is a custom JSON unmarshaler for padding.
+// The field is meaningless bytes, so this is just a passthrough.
+func (p *Padding) UnmarshalJSON(b []byte) error {
```

Comment:
is this necessary?

Reply:
Functionally no, but we need this to be able to assert that the padding is there in the tests